dts/riscv/gigadevice/gd32vf103.dtsi

```diff
@@ -123,6 +123,18 @@
 			label = "I2C_0";
 		};
 
+		spi0: spi@40013000 {
```

Comment:
I think is missing SPI2/3

Reply:
Refer from [GD32 MCUs selection guide](http://gd32mcu.com/data/documents/qitaziliao/GD32_eries_of_MCUs_Selection_Guide(Jan_2022).pdf), gd32vf103 may have 1 or 3 spi interface. This only handle the basic case.